### ISSUE TITLE
Strut online editor for slides

### ISSUE THREAD

**Issue by lightonflux** [libreprojects/libreprojects#110]
http://strut.io/

I know we already have slid.es which is listed as MIT licensed, but i think that only applies to their background technology, not their editor. So one could argue that it isn't FOSS.

Strut on the other side is open source. 

Strut (the editor / authoring tool) is [licensed under the AGPL](https://github.com/tantaman/Strut/blob/master/LICENSE.txt) and it supports impress.js and bespoke.js as slide generators (both FOSS) and also handout.

You can use the hosted service at http://strut.io/ and you don't need to login.

**Comment by yuwash** [comment]
That looks really good and I understand the advantage about more complete FOSS-ness. I just couldn’t try it with Firefox, maybe I’ll try that with Chromium again. IMO it isn’t too bad to have 2 to 3 alternatives with comparable features, if there are significant differences that are beneficial for different users (so we can add it without necessarily replacing slid.es).

**Comment by jancborchardt** [comment]
Yeah, I think it’s ok to have multiple similar things, as long as all of them are well-designed and fit Libre Projects. (Fun fact: I made the Strut logo. ;)
